src/test/java/com/ncc/aif/ScalingTest.java

```diff
@@ -77,20 +77,63 @@
     private int entityIndex = 1;
     private int eventIndex = 1;
     private int assertionIndex = 1;
+    private int clusterCount = 0;
 
-    private static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology";
+    private static final String NAMESPACE = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/LDCOntology";
+    private static final boolean ALLOW_NUMERIC_TYPES = false;
 
     private final Random r = new Random();
     private List<Resource> entityResourceList = null;
 
-    private final String filename = "scalingdata.ttl";
+    private String filename = "scalingdata-" + entityCount + "x" + eventCount + ".ttl";
 
     // Whether to use in memory or disk based.
     // Note:  TDB2 requires transactions, which we do not do!  Do not use it!
     private enum MODEL_TYPE {
         MEMORY, TDB
     }
 
+    // These NIST restrictions correspond to (and should be updated from) com/ncc/aif/ExamplesAndValidationTest.java.
+    private enum NIST_RESTRICTION {
+        RestrictJustification_Edges,    // Edge justification must be aida:CompoundJustification
+        RestrictJustification_Compound, // CompoundJustification must be used only for justifications of argument assertions
+        EdgeJustificationLimit,         // Each edge justification is limited to either one or two spans
+        PreventShotVideo,               // Video must use aida:KeyFrameVideoJustification. Remove ShotVideoJustification
+        FlatClusters,                   // Members of clusters are entity objects, relation objects, and event objects (not clusters)
+        EverythingClustered,            // Entity, Relation, and Event object is required to be part of at least one cluster.
+        ConfidenceValueRange,           // Each confidence value must be between 0 and 1
+        ClusterHasIRI,                  // Entity, Relation, and Event clusters must have IRI
+        JustifyTypeAssertions,          // Each entity/relation/event type statement must have at least one justification
+        NameMaxLength                   // Each entity/filler name string is limited to 256 UTF-8 characters
+    }
+
+    // Tweak these values to introduce more/fewer invalid content in the generated KB.
```

Comment:
I don't have anything against these particular values.  However, I assume this is intended to be a percentage, but the comment isn't clear.  It should indicate what the range of values should be and whether a higher value introduces more invalid content.

Reply:
Good idea-- improved comment.